spyder/app/tests/test_mainwindow.py

```diff
@@ -763,19 +773,20 @@ def test_set_new_breakpoints(main_window, qtbot):
     main_window.editor.clear_all_breakpoints()
     main_window.editor.close_file()
 
-
```

Comment:
Why was this line removed? Two spaces between top-level functions, I thought...accident?

Reply:
yep, ty